both/collections/registration.js

```diff
@@ -16,25 +16,20 @@ Registrations.attachSchema({
   uniqueName: {
     type: String,
     custom: function() {
-      // this.docId is not available to the client on the first validation check
-      // due to a bug. When the server validation hits then the docId makes it
-      // back to the client.
+      // this.docId is not available to the client on the first validation check due to a
+      // bug. When the server validation hits then the docId makes it back to the client.
```

Comment:
I wonder if this comment has rotted. Looks like @FatBoyXPC's PR was accepted: https://github.com/aldeed/meteor-collection2/pull/164.